db/init.sql

```diff
@@ -0,0 +1,3 @@
+CREATE USER postgres;
+ALTER USER postgres WITH SUPERUSER;
+
```

Comment:
Postgres will run this at startup so `FATAL:  role "postgres" does not exist` never happens again 🥳